src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -263,7 +263,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		#else
 		return RiskProvider(
 			configuration: .default,
-			store: store,
+			diaryStore: diaryStore,
```

Comment:
Wait, what? Do we have an extension on `RiskProvider` that allows passing instead of the `SecureStore` the `DiaryStore`?
Otherwise, I don't think that this will work because the `RiskProvider` needs a `SecureStore`, not a `DiaryStore`.
And also, I do not see in this PR why this is need for the `RiskProvider` (because there are no changes in him self).

Reply:
@pascal-brause This happened while automatic renaming. I already fixed it. Thanks for pointing this out.